server/feeders/reviewsgenerator.js

```diff
@@ -107,16 +109,24 @@ var triggerRestaurantsRatings = function() {
   var servicePath;
 
   var q = async.queue(function(task, callback) {
-
+    var restaurantId = utils.asciiEncode(
+      utils.stripForbiddenChars(
+        unescape(task.restaurantName)));
```

Comment:
why don't we use an id which it is not the name of the restaurant? you would avoid many troubles. The code is starting to get complex and you should take into account that this is a tutorial ...